lib/src/core/routing/not_found_screen.dart

```diff
@@ -7,9 +11,19 @@ class NotFoundScreen extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: AppBar(),
       body: Center(
-        child: Text('404 - Page not found!'.hardcoded),
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            Text('404 - Page not found!'.hardcoded),
+            AppGaps.gapH24,
+            TextButton(
+              onPressed: () => Beamer.of(context)
+                  .beamToReplacementNamed("/${context.read<LocaleCubit>().state.locale.languageCode}"),
+              child: const Text('Home'),
+            )
+          ],
+        ),
```

Comment:
Removed the awful app bar and added a text button to redirect the user to home. If the language don¿t match with supported ones, will redirect to EN home

Reply:
Niiice!